app/views/completed_transactions/_fields.html.erb

```diff
@@ -7,4 +7,28 @@
 <% end %>
 
 <%= render partial: 'shared/common_edition_tags', locals: {f: f} %>
+
+<hr />
+<div class="row">
+  <div class="col-md-8 edition-tags" data-module="checkbox-toggle">
+    <h3 class="remove-top-margin">Promote organ donor registration</h3>
+    <p class="help-block add-bottom-margin">
+      Displays the organ donor registration promotion above the completed transaction satisfaction survey
```

Comment:
This copy can be shortened too, the title already indicates the nature of the promotion so here it can just say "promotion":
> Displays a promotion above the satisfaction survey